### ISSUE TITLE
[speech] scene 4 Deaf needs changing to D/deaf

### ISSUE THREAD

**Issue by jade-mc** [w3c/wai-people-use-web-videos#233]
To show the difference between communities.

**Comment by jade-mc** [comment]
This should be changed throughout the scripts. @nitedog are we going to have a final proof/edit? I haven't been commenting on typos as I go, and there are a few places where the structure might need a little tweaking, (for example, to cut down on the use of... for example!) Happy to re-live my proofing/editing days and take this on if needed.

**Comment by nitedog** [comment]
@shawna-slh what are the WAI conventions for writing:

- D/deaf
- Hard of Hearing (HoH)
- DB/deafblind

**Comment by shawna-slh** [comment]
Unfortunately it’s not in the Style Guide, and not consistent throughout the site.

Proposal based on recent usage and discussions:
1. Deaf. So far we have decided that it’s better not to have the complexity of “D/deaf”. Capitalized because more people feel more strongly about it being capitalized.
2. hard-of-hearing. It’s easier for most people to 'parse' with hyphens. Better not to abbreviate it in almost all uses on the W3C site. If _really_ need to abbreviate it because lots of repetitive use of the phrase in short space, then ‘hard-of-hearing (HoH)’ … ‘HoH’.
3. deaf-blind (preferred), deaf-blindness (acceptable). e.g.: …user who is deaf-blind. Persona with deaf-blindness. Hyphenated deaf-blind easier to parse than deafblind not. Don't abbreviate DB. 

I can provisionally add that to the Style Guide and invite comment…

**Comment by bruce-usab** [comment]
+1 for enumerated to plan for https://github.com/w3c/wai-media-guide/issues/196

Fair point about Deaf and not-being-able-to-hear not being the same groups of people.  I do not think <q>D/deaf</q> captures that nuance.  Is scene 4 public facing?

**Comment by jade-mc** [comment]
Isn't that exactly what D/deaf is intended to capture?
This will be public facing in that it'll be in the captions and transcript.

**Comment by bruce-usab** [comment]
> Isn't that exactly what D/deaf is intended to capture?

Yes, but it is a very esoteric convention.  In most contexts, I think it would be more likely to cause confusion rather than showing the difference between communities.

**Comment by shawna-slh** [comment]
> I do not agree that D/deaf captures that nuance, at least not in most settings.

+1

Additionally, we cover it in:
1. https://deploy-preview-113--wai-people-use-web.netlify.app/people-use-web/user-stories-six/
2. https://deploy-preview-113--wai-people-use-web.netlify.app/people-use-web/abilities-barriers-auditory/

straw-proposal:
* Use 'Deaf' for the persona and video
* In 1 & 2 above, consider briefly explaining that some prefer/identify as "Deaf" and some as "deaf". 
* avoid 'D/deaf'. if anything, use 'Deaf/deaf' &mdash; however as mentioned above, that is likely to be confusing without an explanation
